### DISCUSSION TITLE
Multiple connections to so-called Coin Mining Services

### DISCUSSION THREAD

**Discussion by L4N5** [formbricks/formbricks#4355] [General]
Today I got an abuse notification of my server provider for using multiple connections to so-called Coin Mining Services. I noticed in my load that the cpu was pretty high. There were 5 processes running with command ./4 -c/tmp/.. and each of them were using max cpu. My server is only running a few docker containers and after stopping the formbricks container the load instantly went to normal (from 6 to < 0).

Anyone ever experienced this?

I was running the latest container (ghcr.io/formbricks/formbricks:latest) so this should be 2.7.1.

**Comment by mattinannt** [comment]
That's odd. I don't think this is coming from Formbricks.

**Reply by L4N5** [reply]
I agree it's odd. I didn't debug the issue as I'm currently not using the container. So a quick fix for abuse was to stop it for now. I think some bad actor found a way to get in to the container and ran these commands.
